targets/targets.mk

```
@@ -25,5 +25,6 @@ $(eval $(call GluonTarget,x86,64))
 
 ifneq ($(BROKEN),)
 $(eval $(call GluonTarget,bcm27xx,bcm2710)) # BROKEN: Untested
+$(eval $(call GluonTarget,bcm63xx,generic)) # BROKEN: Untested
```

```suggestion
$(eval $(call GluonTarget,bcm63xx,generic)) # BROKEN: No 11s support
```

Broadcom chips are not capable of mesh. Maybe you can test, the client network is working.